media/transcoder/create_job_with_periodic_images_spritesheet.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transcoder
+
+// [START transcoder_create_job_with_periodic_images_spritesheet]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/golang/protobuf/ptypes/duration"
+
+	transcoder "cloud.google.com/go/video/transcoder/apiv1beta1"
+	transcoderpb "google.golang.org/genproto/googleapis/cloud/video/transcoder/v1beta1"
+)
+
+// createJobWithPeriodicImagesSpritesheet creates a job from an ad-hoc configuration and generates two spritesheets from the input
+// video. Each spritesheet contains images that are captured periodically based on a user-defined time interval.
+func createJobWithPeriodicImagesSpritesheet(w io.Writer, projectID string, location string, inputURI string, outputURI string) error {
+	// projectID := "my-project-id"
+	// location := "us-central1"
+	// inputURI := "gs://my-bucket/my-video-file"
+	// outputURI := "gs://my-bucket/my-output-folder/"
+	ctx := context.Background()
+	client, err := transcoder.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("NewClient: %v", err)
+	}
+	defer client.Close()
+
+	req := &transcoderpb.CreateJobRequest{
+		Parent: fmt.Sprintf("projects/%s/locations/%s", projectID, location),
+		Job: &transcoderpb.Job{
+			InputUri:  inputURI,
+			OutputUri: outputURI,
+			JobConfig: &transcoderpb.Job_Config{
+				Config: &transcoderpb.JobConfig{
+					ElementaryStreams: []*transcoderpb.ElementaryStream{
```

Comment:
Optional: you could define separate variables for these parts of the request to possibly make it a bit easier to follow.

Maybe `elementaryStreams`, `muxStreams`, and `spriteSheets`?

Reply:
I feel like this will just add unnecessary lines to the sample.